gibberish_client/src/components/Gameroom/Timer.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
```

Comment:
I guess we can refactor this into
`import React, { useState, useEffect } from 'react'`

and remove `React.` from lines 4 and 5 :)